packages/cohort/src/replicator2/cohort_replicator.rs

```diff
@@ -77,7 +77,7 @@ where
 
     pub async fn commit(&mut self) {
         if let Some(offset) = self.suffix.find_commit_offset(self.latest_in_flight) {
-            if let Err(error) = self.receiver.commit(offset).await {
+            if let Err(error) = self.receiver.commit().await {
```

Comment:
It looks like we are extracting offset from suffix into Some(offset) but then calling receiver.commit() without params. Is this intentional?

Reply:
Yes this is fine, as at the time of commit we were always using the one stored on `tpl (topic partition list)`